website/content/docs/connect/ca/vault.mdx

```diff
@@ -139,8 +139,10 @@ The key after the slash refers to the corresponding option name in the agent con
   path does not exist, Consul will attempt to mount and configure this
   automatically.
 
-  When WAN Federation is enabled, every secondary
-  datacenter must specify a unique `intermediate_pki_path`.
+  When WAN Federation is enabled, every secondary datacenter which shares a common Vault cluster
+  must specify a unique `intermediate_pki_path`. If a Vault cluster is not being used by more than
+  one Consul datacenter, then this is not required. However, it is still recommended to use a
+  unique `intermediate_pki_path` per datacenter for operational and diagnostic clarity.
```

Comment:
```suggestion
  When WAN federation is enabled, every secondary datacenter that shares a common Vault cluster
  must specify a unique `intermediate_pki_path`. If a Vault cluster is not used by more than
  one Consul datacenter, then the intermediate PKI path setting is not required. We still recommend using a
  unique `intermediate_pki_path` for each datacenter, however, to improve operational and diagnostic clarity.
```

Reply:
> If a Vault cluster is not used by more than one Consul datacenter, then the intermediate PKI path setting is not required.

The phrase above is slightly wrong; how does this read?

> If a Vault cluster is not used by more than one Consul datacenter, then **the uniqueness** is not required.

Reply:
It's still kind of awkward. How about:

> If a Vault cluster is not used by more than one Consul datacenter, then you do not need to specify a unique value for the `intermediate_pki_path`. 

Right? You're saying that you still need to configure the setting, but the value doesn't have to be unique, correct?

Reply:
Yeah that works!